### PR TITLE
Logging improvements

### PR DESCRIPTION
**Pull request by MartinBasti**

### CODE REVIEW

**Comment by MartinBasti** [comment]
Thank you, fixed

File: omps/settings.py [hunk 1]
```diff
@@ -78,7 +78,10 @@ class Config(object):
         },
         'log_format': {
             'type': str,
-            'default': '%(asctime)s - %(name)s - %(levelname)s - %(message)s',
+            'default': (
+                '%(asctime)s - [%(process)d] %(name)s - %(levelname)s - '
```
Comment (athos-ribeiro) [inline omps/settings.py]: Since time will be fixed in UTC, would it be useful to append the timezone ti the logs at all, or would that be too verbose?

Comment (MartinBasti) [reply omps/settings.py]: I've updated it to UTC time format.

example: `1994-11-05T13:15:30Z`

Comment (MartinBasti) [reply omps/settings.py]: https://www.w3.org/TR/NOTE-datetime  ISO 8601

**Pull request merged**